ArduSub/AP_Arming_Sub.cpp

```diff
@@ -38,6 +38,13 @@ bool AP_Arming_Sub::has_disarm_function() const {
             }
         }
     }
+    // check if an AUX function that disarms or estops is setup
+    if (rc().find_channel_for_option(RC_Channel::AUX_FUNC::MOTOR_ESTOP) || 
+       rc().find_channel_for_option(RC_Channel::AUX_FUNC::MOTOR_ESTOP) || 
```

Comment:
Got `MOTOR_ESTOP` in here twice. Might also want `MOTOR_INTERLOCK`, although you would have to check if it works for sub.

Reply:
fixed